technical_architecture/performances_guide/batch_page_size.rst

```diff
@@ -0,0 +1,13 @@
+Job product batch size
+======================
+
+A product catalog is unique to one's need. It is configured with a custom number of attributes, families, locales, channels, etc. Those different combinations may affect the import/exports performances.
+
+Indeed the size of the products will define how much RAM they will take to be processed. During a product import/exports, we regularly flush the processed products into the database to avoid too much memory consumption.
```

Comment:
```suggestion
Indeed the size of the products will define how much RAM they will take to be processed. During product import/exports, we regularly flush the processed products into the database to avoid too much memory consumption.
```